ci/docker/Dockerfile.build.android_arm64

```diff
@@ -18,23 +18,20 @@
 #
 # Dockerfile to build MXNet for Android ARM64/ARMv8
 
-FROM ubuntu:16.04 as ccachebuilder
-
-COPY install/ubuntu_core.sh /work/
-RUN /work/ubuntu_core.sh
-COPY install/ubuntu_ccache.sh /work/
-RUN /work/ubuntu_ccache.sh
+#FROM ubuntu:16.04 as ccachebuilder
+#COPY install/ubuntu_core.sh /work/
+#RUN /work/ubuntu_core.sh
+#COPY install/ubuntu_ccache.sh /work/
+#RUN /work/ubuntu_ccache.sh
```

Comment:
Could you elaborate?

Reply:
Let's get the ARMv7 PR merged first and then I ammend this one and re-enable the ccache logic.